test/ads/vpaid/VPAIDIntegrator.spec.js

```diff
@@ -743,9 +744,44 @@ describe("VPAIDIntegrator", function () {
     });
   });
 
+  describe("must support autoResize", function() {
+    it("must handle window resize", function() {
+      var vpaidIntegrator = new VPAIDIntegrator(player, {autoResize: true});
+      var adUnit = new FakeAdUnit();
+      vpaidIntegrator._setupEvents(adUnit, new VASTResponse(), noop);
+      window.dispatchEvent(new Event('resize'));
```

Comment:
It is better if you use "dom.dispatchEvent(window, ....." otherwise the test will not pass on IE8. Since there is no dispatch event on IE8 instead it has fireEvent